lib/contracts.rb

```diff
@@ -72,7 +77,7 @@ def initialize(klass, method, *contracts)
     end
 
     @args_contract_index = args_contracts.index do |contract|
-      contract.is_a? Contracts::Args
+      contract.is_a? Contracts::Builtin::Args
```

Comment:
I think our code is fine using just `Contracts::Args` here and at all other places.